packages/e2e-tests/src/specs/editor/media/insertWebMVideo.js

```diff
@@ -38,6 +38,9 @@ describe('Inserting WebM Video', () => {
   });
 
   async function openA11yPanel() {
+    // open style pane
+    await expect(page).toClick('li', { text: /^Style$/ });
```

Comment:
New sidebar structure doesn't show the design pane (now called the style pane) all the time like before. To expose the panels, the user must click over to it.